algorithms/map_the_debris.js

```diff
@@ -0,0 +1,36 @@
+/*
+https://www.freecodecamp.com/challenges/map-the-debris
+Return a new array that transforms the element's average altitude into their
+orbital periods.
+
+The array will contain objects in the format {name: 'name', avgAlt: avgAlt}.
+
+You can read about orbital periods on wikipedia.
+
+The values should be rounded to the nearest whole number. The body being orbited
+ is Earth.
+
+The radius of the earth is 6367.4447 kilometers, and the GM value of earth is
+398600.4418 km3s-2.
+
+*/
+
+function orbitalPeriod(arr) {
+  var GM = 398600.4418;
```

Comment:
variables which are constants should be declared globally, and the name should be capitalized. So in your code `GM` &  `EARTHRADIUIS` are global constants